### PR TITLE
seed reservations and reservation nights

### PR DESCRIPTION
**Pull request by natevenn**
generate association b/w reservation-user-and-properties
create reservation_nights based off of reservation checkin and checkout dates
closes#63

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@ def initialize
   def generate_admin
     fullname = Faker::Name.name
     email = Faker::Internet.free_email(fullname.split[1])
-    100.times do |i|
+    200.times do |i|
```
Comment (Salvi6God) [inline db/seeds.rb]: stepping it up a notch I feel that!

File: db/seeds.rb [hunk 2]
```diff
@@ -29,7 +29,7 @@ def generate_property_types
   end
 
   def generate_properties
-    user_ids = User.all.pluck(:id)
+    user_ids = User.all.pluck(:id).take(100)
```
Comment (Salvi6God) [inline db/seeds.rb]: you don't need that "#all" I just double checked on my rails console and you can just do 

User.pluck(:id).take(100)

Comment (natevenn) [inline db/seeds.rb]: sweet. thanks for pointing that out

**Comment by Salvi6God** [comment]
Awesome! I'll go ahead and merge now.

**Pull request merged**